examples/example_system/jane/jane_enum.json

```diff
@@ -0,0 +1,9 @@
+{
+    "JANE_fish": {
+        "TUNA": -1,
```

Comment:
What is the rationale behind negative enums?

Reply:
i've seen negative enums in a few applications, might as well support it